### DISCUSSION TITLE
Is there a Windows 7 32-bit version of GDevelop's new release?

### DISCUSSION THREAD

**Discussion by Mythril382** [4ian/GDevelop#2601] [Archive] [answered]
My laptop was now able to install things after reset so I wanted to download GDevelop,is there a Windows 7 32-bit version of GDevelop? I really wanna install it.

**Comment by Silver-Streak** [comment, marked as answer]
> My laptop was now able to install things after reset so I wanted to download GDevelop,is there a Windows 7 32-bit version of GDevelop? I really wanna install it.

Hello! 

While I appreciate your circumstances, GDevelop 5 does not officially support 32 bit OSes. There are some unofficial builds on the forums, although I'm not sure if they're current. Please note that these are as titled, unofficial, and therefore unsupported.
